common/eqemu_logsys.h

```diff
@@ -302,41 +302,41 @@ class EQEmuLogSys {
 		int         instance_id;
 	};
 
-	OriginationInfo origination_info{};
+	OriginationInfo m_origination_info{};
 
 	/**
 	 * Internally used memory reference for all log settings per category
 	 * These are loaded via DB and have defaults loaded in LoadLogSettingsDefaults
 	 * Database loaded via LogSys.SetDatabase(&database)->LoadLogDatabaseSettings();
 	*/
-	LogSettings log_settings[Logs::LogCategory::MaxCategoryID]{};
+	LogSettings m_log_settings[Logs::LogCategory::MaxCategoryID]{};
 
 	struct DiscordWebhooks {
 		int         id;
 		std::string webhook_name;
 		std::string webhook_url;
 	};
 
-	DiscordWebhooks discord_webhooks[MAX_DISCORD_WEBHOOK_ID]{};
+	DiscordWebhooks m_discord_webhooks[MAX_DISCORD_WEBHOOK_ID]{};
 
-	bool file_logs_enabled = false;
+	bool m_file_logs_enabled = false;
 
-	int         log_platform = 0;
-	std::string platform_file_name;
+	int         m_log_platform = 0;
+	std::string m_platform_file_name;
```

Comment:
I'm generally only a fan of prefixing private variables since public access is on an object and doesn't need to distinguish from locals, how much of these be moved to private?

Reply:
I can get on board with that, made the easier ones private that should have been private anyways.

Kept 1-2 public for now